docs/create-path-across-chain.md

```diff
@@ -32,7 +32,9 @@ In our "Relaying Packets Across Chains" example, we set up the relayer to relay
     - `rly transact connection`
     - `rly transact channel`
 
-    <br>
+   <br>
+   
+    One can add the ` --version "{\"fee_version\":\"ics29-1\",\"app_version\":\"ics20-1\"}" ` flag to the `link` transaction in order to create a channel that supports the [ICS-29](https://github.com/cosmos/ibc/issues/578) fee protocol standard.
```

Comment:
Can we change the hyperlink to reference the ics-29 spec instead of the pr?
https://github.com/cosmos/ibc/tree/main/spec/app/ics-029-fee-payment